src/account-info.js

```diff
@@ -23,6 +23,17 @@ function AccountInfo (object) {
   this._isMobileVerified = Boolean(object.sms_verified);
 
   this._currency = object.currency;
+
+  var notifications = {};
+  if (object.notifications_type) {
+    object.notifications_type.forEach(function (code) {
+      var type = Math.log2(code);
+      if (type === 0) notifications.email = true;
+      if (type === 2) notifications.http = true;
+      if (type === 5) notifications.sms = true;
```

Comment:
Ok. Just noting here that this will change "off" values from undefined to false.